### DISCUSSION TITLE
Supervisor: Create Toast to tell user HA is upgrading

### DISCUSSION THREAD

**Discussion by balloob** [home-assistant/frontend#11206] [Other feature requests]
**Is your feature request related to a problem? Please describe.**
Once you upgrade Home Assistant, you should wait till the upgrade is complete and not start other actions. The frontend should reload as chunks for code splitted panels that work with the current loaded frontend are no longer available. 

**Describe the solution you'd like**
A blocking modal showing we're doing things. Preferably a bit more than just a spinner but for MVP it's fine. Once update complete, hard refresh the frontend so we get latest frontend. Preferably on every tab that's open. 

**Describe alternatives you've considered**
Just show a small card that people forget they clicked until UI stops working. 

**Additional context**

**Comment by bramkragten** [comment]
Should we make this a toast? So you can still use the page while the update is downloading?

**Comment by balloob** [comment]
That's a good idea

**Comment by matthiasdebaat** [comment]
Right now Home Assistant got a blocking model. Should this issue be closed?

But before you do that. What is the reason the user shouldn't navigate?  When I reload the page i can navigate again. Should this dialog also appear when loading different pages?

**Comment by ludeeus** [comment]
That's why it's still open.

**Comment by matthiasdebaat** [comment]
OK. Do you know what's the reason to block HA?

**Comment by ludeeus** [comment]
That's described in the OP, it's going down, you should not do anything until it's finished
